### DISCUSSION TITLE
.NET MAUI support

### DISCUSSION THREAD

**Discussion by rafalka** [reactiveui/ReactiveUI#2790] [General]
Hi ReactiveUI team,

I'm wondering when you'll add support for .NET MAUI?
Can we expect it will be available close to the official MAUI release (November 2021)?

Best Regards,
Rafal

**Comment by glennawatson** [comment]
As soon as they release some NuGet packages to the nuget.org rather than to the azure devops server support will be added. Just waiting on the team.

**Comment by rafalka** [comment]
so, assuming they'll do that in Nov, any aprrox. date when we can expect ReactiveUI?
I know that it is hard to estimate right now, but what's your gut feeling: It will be 1,  3 or rather 6 months after?

We starting a new project and we'd like to migrate to MAUI as soon as possible (desktop is one of our targets). 
This kind of information would really help me planning a project milestones.

**Reply by glennawatson** [reply]
Be fairly quick, like 1 month.

**Comment by rafalka** [comment]
Cool! :)

Thanks for prompt answers.

**Comment by ChrisPulman** [comment]
Just in case you hadn't spotted the new ReactiveUI.Maui package, it's now available.

**Reply by ooikengsiang** [reply]
I tried to add ReactiveUI.Maui / ReactiveUI nuget package to a newly created MAUI project in VS 2022 Preview 6, but nuget manager seem unhappy with the target framework:
Invalid restore input. The original target frameworks value must match the aliases. Original target frameworks: net6.0-ios;net6.0-android;net6.0-maccatalyst;net6.0-windows10.0.19041;net6.0-windows10.0.19041, aliases: net6.0-ios;net6.0-android;net6.0-maccatalyst;net6.0-windows10.0.19041.

**Comment by glennawatson** [comment]
Have you installed the correct workloads using the dotnet command? https://docs.microsoft.com/en-us/dotnet/maui/get-started/installation

**Reply by ooikengsiang** [reply]
I tested on a newly created MAUI project that is able to compile successfully and deploy to Windows (required csproj modification) and iOS simulator without problem before trying to add ReactiveUI.MAUI package from nuget.
MAUI is still pretty buggy in this preview, thus I'm not sure the error is MAUI, nuget target framework version or ReactiveUI target framework version.

**Comment by warpedgeoid** [comment]
Just curious if any plans exist to update ReactiveUI.Maui now that RC1 bits are available for Maui? As of Maui P14, these things must be linked against Microsoft.iOS instead of Xamarin.iOS, but breaking changes going forward should be minimal.

**Reply by anaisbetts** [reply]
Fam if you want something don't ask "What is your Enterprise Roadmap", send a PR!

**Reply by glennawatson** [reply]
I been waiting on rc1 since the preview was breaking with Uno and now time to get a PR together
